docs/source/basics/writing_dockerfiles.md

```diff
@@ -47,6 +47,11 @@ WORKDIR /root
 _Note that we must use the base image specified in the first line to configure
 libraries and settings for consistent task behavior._
 
+There are 3 latch-base images available as seen on [github](https://github.com/latchbio/latch-base):
```

Comment:
base images change frequently enough that i dont think you should hard code them into the documentation - just linking to the github is enough